stack.go

```diff
@@ -87,7 +92,9 @@ func backtraceFromErrorWithStackTrace(e stackTracer) []StackFrame {
 			Func: fmt.Sprintf("%n", f),
 		}
 	}
-	return frames
+
+	pkgName, _ := packageFuncName((uintptr)(stackTrace[0]))
```

Comment:
there is no need to case stacktrace to uinptr - use public API documented by pkg/errors